zlibWrapper/examples/zwrapbench.c

```diff
@@ -130,7 +130,7 @@ void BMK_SetBlockSize(size_t blockSize)
 **********************************************************/
 typedef struct
 {
-    const char* srcPtr;
+    char* srcPtr;
```

Comment:
I see the warning now on OS-X `clang` : 
`warning: assigning to 'z_Bytef *' (aka 'unsigned char *') from 'const void *' discards qualifiers`

That's strange, because `next_in` is supposed to be `z_const Bytef *`, no just `Bytef *`.

So, for some reason, `z_const` is somewhat translated into nothing.

`z_const` is defined into `zconf.h` (which is included by `zlib.h`) : 
```
#if defined(ZLIB_CONST) && !defined(z_const)
#  define z_const const
#else
#  define z_const
#endif
```

A solution could be to `#define ZLIB_CONST` 
or even directly `#define z_const const`
before including `zlib.h` .

It's unclear why this design choice was made.
Looks like a design error to me.
A `const` property should not be optional.